Source/Core/Core/PowerPC/JitCommon/Jit_Util.cpp

```diff
@@ -18,6 +18,26 @@ using namespace Gen;
 
 void EmuCodeBlock::MemoryExceptionCheck()
 {
+  // TODO: We really should untangle the trampolines, exception handlers and
+  // memory checks.
```

Comment:
There are still two callers in `Jit_LoadStoreFloating` that require `MemoryExceptionCheck` because they call `WriteToConstAddress` instead of `SafeWriteRegToReg`. Those are gone in the regcache code.